appinventor/components/src/com/google/appinventor/components/runtime/ListView.java

```diff
@@ -26,19 +26,18 @@
 import android.widget.LinearLayout;
 
 import android.widget.LinearLayout.LayoutParams;
-import com.google.appinventor.components.annotations.DesignerComponent;
-import com.google.appinventor.components.annotations.DesignerProperty;
-import com.google.appinventor.components.annotations.IsColor;
-import com.google.appinventor.components.annotations.PropertyCategory;
-import com.google.appinventor.components.annotations.SimpleEvent;
-import com.google.appinventor.components.annotations.SimpleObject;
-import com.google.appinventor.components.annotations.SimpleProperty;
+import com.google.appinventor.components.annotations.*;
```

Comment:
We use single imports for App Inventor, not wildcard imports.

Reply:
Fixed now